analyzer/codechecker_analyzer/analyzers/config_handler.py

```diff
@@ -118,7 +118,7 @@ def checks(self):
         """
         return self.__available_checkers
 
-    def __gen_name_variations(self):
+    def __gen_name_variations(self, only_prefix=False, unique=False):
```

Comment:
When is it useful getting non-unique results?

Reply:
Previously, it was necessary to collect all name variations. I just didn't want to create a new backward incompatibility function. But it is ok for me to have just unique mode.